src/CMakeLists.txt

```diff
@@ -78,7 +78,15 @@ if(NOT HAVE_REGEX)
   list(APPEND sources regexp.c)
 endif()
 
-add_library(rdkafka SHARED ${sources})
+option(RDKAFKA_BUILD_STATIC "Build static rdkafka library" OFF)
```

Comment:
Can you add instructions for how to build static libraries to packaging/cmake/README.md?

Reply:
now ok?